lib/mix/tasks/hex.outdated.ex

```
@@ -207,6 +209,20 @@ defmodule Mix.Tasks.Hex.Outdated do
     end
   end
 
+  defp maybe_sort_by(values, "status") do
+    status_order = %{
+      "Up-to-date" => 1,
+      "Update not possible" => 2,
+      "Update possible" => 3
+    }
+
+    Enum.sort_by(values, fn [_package, _lock, _latest, [_color, status]] ->
+      Map.fetch!(status_order, status)
+    end)
+  end
+
+  defp maybe_sort_by(values, _), do: values
```

This should have been `defp maybe_sort_by(values, nil)`, in other words, we shouldn't silently ignore incorrect sort-by value. I fixed it on main: f4e76b6.

Thank you for the PR, this is a really nice feature!

---

Great, thanks Wojtek!